pkg/registry/core/service/strategy_test.go

```diff
@@ -1030,30 +1030,6 @@ func TestPatchAllocatedValues(t *testing.T) {
 		update: svctest.MakeService("foo",
 			svctest.SetTypeClusterIP),
 		expectSameClusterIPs: true,
-	}, {
-		name: "first_IP_patched_because_policy",
```

Comment:
I couldn't tell. But i assume these tests moved somewhere else?

Reply:
These specific cases go away because we later truncate - it made more sense in the same code that does all the other IP logic